src/fm/cp_dlaf_utils.c

```diff
@@ -0,0 +1,49 @@
+/*----------------------------------------------------------------------------*/
+/*  CP2K: A general program to perform molecular dynamics simulations         */
+/*  Copyright 2000-2023 CP2K developers group <https://cp2k.org>              */
+/*                                                                            */
+/*  SPDX-License-Identifier: GPL-2.0-or-later                                 */
+/*----------------------------------------------------------------------------*/
+
+#if defined(__DLAF)
+
+#include <dlaf_c/eigensolver/eigensolver.h>
+#include <dlaf_c/factorization/cholesky.h>
+#include <dlaf_c/init.h>
+
+void dlaf_init() {
+  const char *pika_argv[] = {"cp2k", "--pika:print-bind"};
```

Comment:
Just a reminder that we can remove the `--pika:print-bind` once we have the environment variable in 0.20.0.

Reply:
`PIKA_PRINT_BIND` is available since https://github.com/pika-org/pika/pull/828.  https://github.com/pika-org/pika/pull/834 add an helper script (from @msimberg) with added support to set this variable.

Removing hard coded `--pika:print-bind` since it is no longer necessary. `pika@0.20.0` will be released soon.